app/assets/graphing/graph.max.js

```diff
@@ -170,6 +170,7 @@ var intervalID = setInterval(function() {
                 if (legend.length) {
                     legend = legend[0];
                     legend.style.opacity = data.lineOpacities[yID];
+                    legend.style.stroke =  data.barColors[yID][0];

```

Comment:
[This](https://github.com/dimagi/commcare-android/blob/master/app/assets/graphing/graph.max.js#L147-L149) makes me think `barColors` isn't always defined and you should check for it.